.github/workflows/draft-release.yml

```diff
@@ -0,0 +1,43 @@
+---
+name: "Draft Release"
+
+on:
+  push:
+    tags:
+      - "*"
+
+jobs:
+  draft-release-notes:
+    needs: build-asbackup-mac
+    runs-on: ubuntu-latest
+    name: Draft Release
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+    - name: Get Latest Tag
+      run: |
+        LATEST_TAG=$(git describe --tags `git rev-list --tags --max-count=1`)
+        echo "LATEST_TAG=$LATEST_TAG" >> $GITHUB_ENV
+    # below steps are for downloading release artifacts
+    - name: Download Asbackup Mac Artifact
```

Comment:
Not sure we should upload any artifacts unless we provide them all. Seems weird to me.  Plus, these are unnotorized

Reply:
all meaning "mac, deb, rpm"

Reply:
will remove for now.